### ISSUE TITLE
Currently only works on python>=3.9

### ISSUE THREAD

**Issue by CoderJoshDK** [CoderJoshDK/WIOpy#4]
Currently, the API does not work for versions before python 3.9.

If you try to use an earlier version, you will get a `TypeError: 'type' object is not subscriptable`.  

The issue comes from [wiopy/arguments.py](../blob/master/wiopy/arguments.py#L8):
```py
def get_items_ids(items: Union[str, list[str]]) -> list[str]:
    if not isinstance(items, str) and not isinstance(items, list):
        raise InvalidParameterException('Invalid items argument, it should be a string or List of strings')
    ...
```
`list` type casting only works for python>=3.9  
To fix, import `List` 

----------------
This issue will be fixed. I (the developer) am a little busy. But I will fix this issue and update pypi

**Comment by CoderJoshDK** [comment]
[This commit](https://github.com/CoderJoshDK/WIOpy/commit/19da333d8a3194b85bb6cd0e97099c3faf4eb290) fixes the issues. It will be a bit before PyPI is updated with a new version. 
This issue was brought up by Issue #3

**Comment by CoderJoshDK** [comment]
# Why has this not been done yet? It should be an easy fix, right?
-----
Easy? Well, yes, but actually no.

**tldr; I am busy and do still plan to update things**

The issue is that my method of uploading to PyPI sucks. I hate it. And there are other little changes that I want to make to this package. This issue is minor and will be fixed in the next update. But it requires changes that I do not have experience in. Meaning it requires time. Time is something I am a little short on. This is a low priority. But once I have some time to experiment with the changes I want, I will do that and close this issue. 

Thank you for your patience and understanding!
